.circleci/config.yml

```diff
@@ -64,6 +64,46 @@ jobs:
           key: dependency-cache-{{ checksum "package.json" }}
           paths:
             - node_modules
+
+  test-and-build-management:
+    working_directory: ~/3id-connect/management
+    docker:
+      - image: circleci/node:lts
+    steps:
+      - checkout:
```

Comment:
looks like path not needed here or above, since the working directory defines where all these commands will run

Reply:
My understanding is that the working directory is where the commands are run (here the `management` folder), but we want the code to be checked out to the parent folder, no?